src/CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
-project(minetest)
+project(luanti)
```

Comment:
isn't this redundant actually?

Reply:
e0b6668d6f711fcc51e3e81be4a4781aead9f7ac

Reply:
i guess that's for another day :shrug: 
because we already have a `project()` call in the top-level file